tensorflow/python/keras/engine/network.py

```diff
@@ -1355,6 +1355,10 @@ def save(self, filepath, overwrite=True, include_optimizer=True):
     ```
     """
     if not self._is_graph_network:
+      logging.warning(
+          'Currently `save` requires model to be a graph network. Consider '
+          'using `save_weight`, in order to save the weights of the model.')
+
       raise NotImplementedError
```

Comment:
I'm afraid the warning will get buried. Maybe `NotImplementedError("message")` instead of `logging.warning("message")`? Or `logging.error`.